### PR TITLE
added logo option to powered_by()

### PR DESCRIPTION
**Pull request by rabiibouhestine**
Added the option to show company logo when using `powered_by`.

Added Appsilon logo to /www directory of the example app and used it in the example app.

### CODE REVIEW

File: R/display.R [hunk 1]
```diff
@@ -65,9 +65,11 @@ display <- function(message, position = "top right", type = "message") {
...
 #' @export
 #' @importFrom shiny a p
-powered_by <- function(company_name, link = "#", position = "top right") {
-  display(p(style = "margin: 0;", "Powered by ",
-            a(href = link, target = "_blank", company_name)),
+powered_by <- function(company_name, link = "#", position = "top right",
+                       logo = NULL, logo_width = "120px", logo_height = "auto") {
+  display(div(p(style = "margin: 0;", "Powered by ",
+                a(href = link, target = "_blank", company_name)),
+              img(src=logo, width = logo_width, height = logo_height)),
```
Comment (jakubnowicki) [inline R/display.R:72]: As CI suggests, please add spaces around `=`.

File: R/display.R [hunk 2]
```diff
@@ -65,9 +65,11 @@ display <- function(message, position = "top right", type = "message") {
 #' @return div with "powered by".
 #' @export
 #' @importFrom shiny a p
-powered_by <- function(company_name, link = "#", position = "top right") {
-  display(p(style = "margin: 0;", "Powered by ",
-            a(href = link, target = "_blank", company_name)),
+powered_by <- function(company_name, link = "#", position = "top right",
+                       logo = NULL, logo_width = "120px", logo_height = "auto") {
```
Comment (jakubnowicki) [inline R/display.R:69]: Please add `logo` argument to the documentation.

File: R/display.R [hunk 3]
```diff
@@ -65,9 +65,11 @@ display <- function(message, position = "top right", type = "message") {
...
+  display(div(p(style = "margin: 0;", "Powered by ",
+                a(href = link, target = "_blank", company_name)),
```
Comment (jakubnowicki) [inline R/display.R:71]: Can you make logo clickable? Also, the inline option would be a cool feature here.

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
Good job, but needs some additional work.

File: R/display.R (hunk 1 above)
Comment (rabiibouhestine) [reply R/display.R]: done

File: R/display.R (hunk 2 above)
Comment (rabiibouhestine) [reply R/display.R:72]: done

File: R/display.R (hunk 3 above)
Comment (rabiibouhestine) [reply R/display.R:74]: I made the logo clickable (links to the same link put in the argument `link`).

Comment (rabiibouhestine) [reply R/display.R]: Also added inline option.

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
That is a nice improvement!
Please check my comments in the code.
Also, CI throws an error - this is because you have reduced code coverage below the acceptable threshold (1%). Please add some unit tests for powered_by function.
Last thing - the version that exists on master is already on CRAN. I've created a develop branch so please change the destination of this PR from master to develop. Also, it would be good to change the package version in the DESCRIPTION file. My suggestion here is to add .9000 at the end. This is not a large change so we can treat this as a middle version before a new release.

File: examples/app.R [hunk 4]
```diff
@@ -9,7 +9,7 @@ shinyApp(
   ui = tagList(
     shiny.info::info_panel(
       shiny.info::git_info(),
-      shiny.info::powered_by("Appsilon", link = "appsilon.com"),
+      shiny.info::powered_by("Appsilon", link = "appsilon.com", logo = "appsilon.png"),
```
[the remark this answers is not in the document]
Comment (rabiibouhestine) [reply examples/app.R]: done

File: R/display.R [hunk 5]
```diff
@@ -61,15 +61,32 @@ display <- function(message, position = "top right", type = "message") {
...
+    style_inline <- "display: inline-block;"
+  } else {
+    style_inline <- ""
+  }
+  display(
+    div(
+      p(style = paste0("margin: 0;", style_inline),
+        "Powered by ",
+        a(href = link, target = "_blank", company_name)),
+      a(style = paste0(style_inline),
```
[the remark this answers is not in the document]
Comment (rabiibouhestine) [reply R/display.R]: done

File: R/display.R [hunk 6]
```diff
@@ -61,15 +61,37 @@ display <- function(message, position = "top right", type = "message") {
...
-  display(p(style = "margin: 0;", "Powered by ",
-            a(href = link, target = "_blank", company_name)),
-          position,
-          type = "powered_by")
+powered_by <- function(company_name = NULL, link = "#", position = "top right",
+                       logo = NULL, logo_width = "120px", logo_height = "auto", inline = FALSE) {
+  if (is.null(company_name) & is.null(logo)) {
+    stop("company_name and logo can't be both NULL")
+  }
+  if (inline) {
```
Comment (dokato) [inline R/display.R:77]: or shorter with `ifelse`

**Review by dokato — APPROVED** [review]
LGTM

**Pull request merged**